### DISCUSSION TITLE
road feature description

### DISCUSSION THREAD

**Discussion by rsun0013** [christianbirchler-org/sdc-scissor#81] [Q&A] [answered]
hi, 
full road diversity is one of the features that is extracted, but what does it mean exactly? i cant find any descriptions for it in the sdc-scissor papers.
Thanks

**Comment by ChristianBirchler** [comment, marked as answer]
Hi @rsun0013,

thanks for mentioning this. Indeed, road diversity is a new feature we added for a paper that is under review and has not been published yet. Road diversity is the area spawned between the direct connections from the start and end points of each segment and the interpolated actual road path. However, I created a new issue #82 to add a description of the features to the documentation.

Could I help you with my answer?

Best,
Christian

**Reply by rsun0013** [reply]
thanks, so what does it actually aim to measure in terms of the challenging the av?

**Reply by ChristianBirchler** [reply]
Hi @rsun0013,

Basically, the road diversity feature is an indicator of how simple or just straight the road is. For example, if the road is just straight from the beginning to the end, then the road diversity feature has a very low score. The area between the road and the imaginary direct line from the start to the end of a segment is very small. If the road has many turns and is not straight at all then the areas between the road and the imaginary direct line of the segments' start and end points are larger. Thus, for diverse roads, we get a higher road diversity score. In summary, we can say that the road diversity feature is also a way to say how straight or not straight a road is.

Could I answer your question?

Cheers,
Christian
